src/plugins/wizard/public/application/components/data_tab/dropbox.tsx

```diff
@@ -126,6 +133,7 @@ const DropboxComponent = ({
                 aria-label="clear-field"
                 iconSize="s"
                 onClick={() => onAddField()}
+                data-test-subj="dropBoxAddBtn"
```

Comment:
NIT: i see current standard has `camelButton` or `kebab-button`

Reply:
Our styleguide prescribes `cameCase` with `-` for suffixing the subject with an id of sorts. I've followed that pattern here.